pandas/tests/test_resample.py

```diff
@@ -3400,3 +3400,10 @@ def test_aggregate_with_nat(self):
 
             # if NaT is included, 'var', 'std', 'mean', 'first','last'
             # and 'nth' doesn't work yet
+
+    def test_repr(self):
+        result = repr(TimeGrouper(key='A', freq='H'))
+        expected = ("TimeGrouper(key='A', freq=<Hour>, sort=True, "
+                    "closed='left', label='left', how='mean', axis=0, "
+                    "convention='e', base=0)")
+        assert result == expected
```

Comment:
a better test is to actually construct the object and compare to the original

Reply:
I don't understand you here.

Reply:
no, use the repr to actually construct a new object (that's the point of a repr), and compare to the original

Reply:
I still don't understand. Should the ``expected`` be wrapped in a ``repr()``.